config/home.ts

```diff
@@ -0,0 +1,27 @@
+export const homeData = {
+    tagline: 'One Ecosystem for all Clubs.',
+    description:
+    'Empower your MIT Bengaluru club experience with our centralized platform, seamlessly managing and collaborating on events, blogs, and administration.',
+    FAQConfig : [
```

Comment:
we should also consider making a type for `FAQConfig` (maybe `FAQItem`) so that this array can satisfy `FAQItem[]`.